.github/workflows/playwright_tests.yml

```diff
@@ -0,0 +1,57 @@
+name: Playwright Tests
+
+on: 
+  push:  
+    branches: '**'
```

Comment:
<img width="896" alt="Screenshot 2023-08-02 at 10 01 04 PM" src="https://github.com/WalletConnect/web3modal/assets/966091/5dc86f3e-213e-457d-8ac0-599b6a490338">

I think this causes the tests to runs twice upon PR. Can you narrow down to `V2` and `V3`?

Reply:
You're right, I actually had them swapped, fixed 👍